app/screens/post_options/options/follow_option.tsx

```diff
@@ -40,8 +42,15 @@ const FollowThreadOption = ({thread}: FollowThreadOptionProps) => {
         }
     }
 
+    const unFollowThread = useCallback(() => {
```

Comment:
```suggestion
    const undo = useCallback(() => {
```

also follows the toggle functionality